### PR TITLE
feat: Move time calculation to MDW

### LINKED ISSUE #114: Move time calculation to MDW
### Context information
Some adapters are using block height to calculate time, But some needs to be calculated because there is no time entry in response

### Steps to reproduce

see usage formatBlockDiffAsDatetime in [adapters.js](https://github.com/aeternity/aescan/blob/develop/src/utils/adapters.js)

### What is expected?

Move other calculations to backend, remove `formatBlockDiffAsDatetime()` then

This Issue is followup of https://github.com/aeternity/aescan/issues/100

MDW Tracking https://github.com/aeternity/ae_mdw/issues/831

### PR DESCRIPTION
**Pull request by janmichek**
## Description
partly #114 

## Checklist:

- [x] I have read and followed the [Contributing Guide](../../CONTRIBUTING.md)  
- [x] move some calculations to backend

### CODE REVIEW

File: src/components/OraclesTable.vue [hunk 1]
```diff
@@ -40,11 +40,11 @@
         <td class="oracles-table__data">
           <div>
             <app-link
-              :to="`/keyblocks/${oracle.activeFromHeight}`">
-              {{ oracle.activeFromHeight }}
+              :to="`/keyblocks/${oracle.registeredHeight}`">
+              {{ oracle.registeredHeight }}
```
Comment (janmichek) [inline src/components/OraclesTable.vue:44]: renamed for consistency

File: src/stores/oracleDetails.js [hunk 2]
```diff
@@ -42,7 +39,7 @@ export const useOracleDetailsStore = defineStore('oracleDetails', () => {
   }
 
   async function fetchOracle() {
-    const { data } = await axios.get(`${MIDDLEWARE_URL}/v2/oracles/${oracleId.value}`)
+    const { data } = await axios.get(`${MIDDLEWARE_URL}/v2/oracles/${oracleId.value}?tx_hash=true`)
```
Comment (janmichek) [inline src/stores/oracleDetails.js:42]: as advised in here https://github.com/aeternity/ae_mdw/issues/1425#issuecomment-1700899383

**Review by michele-franchi — APPROVED** [review]
Good job 👏

**Review by lukeromanowicz — APPROVED** [review]
Looks great! I think we should involve @Liubov-crypto to help us carefully test if there are no regressions

**Comment by janmichek** [comment]
@Liubov-crypto please help us to test. The pages needs to be checked you will know by
- you go by looking at changes in `adapters.js`
- it should be nicely named so eg `adaptStateChannelDetails` stands for state channels details. 
- check the affected pages, there should be all values present. Especially this datetime ones.

Thank you

**Comment by Liubov-crypto** [comment]
There is a partly translation time related data on some pages in Chrome browser, however I haven't applied it. Is this acceptable? 
auc
nam
or1
orac

**Comment by janmichek** [comment]
> @janmichek There is a partly translation time related data on some pages in Chrome browser, however I haven't applied it. Is this acceptable? auc nam or1 orac

It's a open topic we will solve later. I think this was the same before the change, right? Could you check?

**Comment by Liubov-crypto** [comment]
@janmichek yes, I see the same on the main branch.

**Pull request merged**